Updated GCP guide (#63)
rapidsai/spark-examples: getting-started-guides/csp/gcp/gcp.md

* `EXCLUSIVE_PROCESS` must be set for all GPUs in each NodeManager.
* `spark.dynamicAllocation.enabled` must be set to False for spark

> There are four steps to run a Sample XGBoost4j app on a GCP GPU Cluster:

Reviewer: Can we change "Sample XGBoost4j" to "sample XGBoost4j"
```suggestion
There are four steps to run a sample XGBoost4j app on a GCP GPU Cluster:
```

1. Create a GPU Cluster with preinstalled GPU Drivers, Spark RAPIDS libraries and Spark XGBoost libraries
2. Copy sample data in to GCS Bucket 
3. Prepare sample XGBoost apps on your local machine. 

> 4. Submit sample PySpark or Scala App using gcloud command from your local machine

Reviewer: ```suggestion
4. Submit sample PySpark or Scala App using the gcloud command from your local machine
```

### Step 1.  Create a GPU Cluster in GCP

> Before you create a cluster, please gitclone the spark-examples directory to your local machine. From the spark-examples directory, copy the spark-gpu initialization scripts inside `spark-gpu` folder into an accessible GCS bucket with following structure:

Reviewer: ```suggestion
Before you create a cluster, please git clone the spark-examples directory to your local machine. From the spark-examples directory, copy the spark-gpu initialization scripts from the `spark-gpu` folder into an accessible GCS bucket with following structure:
```

> ```bash

Reviewer: I recommend explicitly calling out the command:
```
export STORAGE_BUCKET=my_bucket
cd getting-started-guides/csp/gcp/spark-gpu
gsutil cp -r rapids.sh internal gs://$STORAGE_BUCKET
```

    /$STORAGE_BUCKET/spark-gpu/rapids.sh
    /$STORAGE_BUCKET/spark-gpu/internal/install-gpu-driver-ubuntu.sh
    /$STORAGE_BUCKET/spark-gpu/internal/install-gpu-driver-debian.sh
```  

Using the `gcloud` command create a new cluster with Rapids Spark GPU initialization action. The following command will create a new cluster named `<CLUSTER_NAME>` under your Project space. Here we use Ubuntu since that is our recommended OS for Spark-XGBoost on GCP.

> ```bash

Reviewer: Is there a reason we are initializing with Zeppelin?

        --subnet=default \
        --properties 'spark:spark.dynamicAllocation.enabled=false,spark:spark.shuffle.service.enabled=false' \
        --enable-component-gateway
```

> This cluster is now ready to host RAPIDS Spark XGBoost Applications.

Reviewer: ```suggestion
This cluster is now ready to host RAPIDS Spark XGBoost applications.
```
